src/OpenTelemetry.Resources.OperatingSystem/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## 0.1.0-alpha.2
+
+Released 2024-Jul-17
```

Comment:
These are added [automatically](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1950) by the prepare release workflow.